fixtures/stringify.css

```diff
@@ -9,6 +9,26 @@
     }
 }
 
+@import 'foo' layer(bar);
+
+@layer foo {
+    .class {
+        color: red;
+    }
+
+    @layer bar {
+        .class2 {
+            color: red;
+        }
+    }
+}
+
+@layer . {
```

Comment:
When I wrote about a layer name with a dot, I meant something like `foo.bar`. A single dot is incorrect layer name. Also a layer name can be omitted, good to add such a case as well.
https://drafts.csswg.org/css-cascade-5/#layer-names
<img width="819" alt="image" src="https://github.com/user-attachments/assets/ebda7f67-38c4-43da-9a0b-d1a506c034a2">

https://drafts.csswg.org/css-cascade-5/#unnamed-layers
<img width="840" alt="image" src="https://github.com/user-attachments/assets/003ec601-7aba-4797-935a-3d4e09f2bced">

Reply:
Ah gotcha.